ui/page/overview/overview_layout.go

```diff
@@ -376,9 +393,43 @@ func (pg *AppOverviewPage) listenForSyncNotifications() {
 				case wallet.SyncCompleted:
 					pg.loadTransactions()
 					pg.RefreshWindow()
-				case wallet.BlockAttached:
+				}
+
+			case n := <-pg.TxAndBlockNotifChan:
+				switch n.Type {
+				case listeners.NewTransaction:
+					pg.loadTransactions()
+					pg.RefreshWindow()
+				case listeners.BlockAttached:
 					pg.RefreshWindow()
 				}
+			case n := <-pg.ProposalNotifChan:
+				switch n.ProposalStatus {
+				case wallet.Synced:
+					pg.loadRecentProposals()
+					pg.RefreshWindow()
+				}
+
+			case n := <-pg.BlockRescanChan:
+				if n.Stage == wallet.RescanEnded {
```

Comment:
Missing a `pg.rescanUpdate = &n`:
```suggestion
			case n := <-pg.BlockRescanChan:
				pg.rescanUpdate = &n
				if n.Stage == wallet.RescanEnded {
```